src/custom-elements/src/components/carousel/carousel.tsx

```diff
@@ -225,8 +225,14 @@ export class Carousel {
     this.chiViewChange.emit(view);
   }
 
+  _checkMultipleItems() {
+    const carouselItems = this.el.querySelectorAll(`.${CAROUSEL_CLASSES.ITEM}`);
+
+    return carouselItems.length > 1;
+  }
```

Comment:
Since this will return a boolean value, it is more readable in my opinion to have something like _areThereMultipleItems_, or similar. Thoughts?

Reply:
I am agree, I think it ismore readable.